src/LocalDatastore.js

```diff
@@ -313,15 +360,14 @@ const LocalDatastore = {
       await Promise.all(pinPromises);
       this.isSyncing = false;
     } catch(error) {
-      console.log('Error syncing LocalDatastore'); // eslint-disable-line
-      console.log(error); // eslint-disable-line
+      console.error('Error syncing LocalDatastore: ', error);
```

Comment:
do all browsers support console.error (that's a question, I dunno).

Reply:
yeah same as warn and log